### PR TITLE
play sounds when waxing sign & when interacting with a waxed sign

### PR DESCRIPTION
**Pull request by Erkam246**

### CODE REVIEW

**Review by RestartFU — APPROVED** [review]
looks good, I think we can merge this

File: server/block/sign.go [hunk 1]
```diff
@@ -119,10 +120,13 @@ func (s Sign) Wax(cube.Pos, mgl64.Vec3) (world.Block, bool) {
 }
 
 // Activate ...
-func (s Sign) Activate(pos cube.Pos, _ cube.Face, _ *world.World, user item.User, _ *item.UseContext) bool {
+func (s Sign) Activate(pos cube.Pos, _ cube.Face, w *world.World, user item.User, _ *item.UseContext) bool {
 	if editor, ok := user.(SignEditor); ok && !s.Waxed {
 		editor.OpenSign(pos, s.EditingFrontSide(pos, user.Position()))
 	}
+	if s.Waxed {
```
Comment (Sandertv) [inline server/block/sign.go:127]: Could change this to `else if s.Waxed`.

File: server/world/sound/block.go [hunk 2]
```diff
@@ -176,6 +176,12 @@ type ComposterReady struct{ sound }
 // LecternBookPlace is a sound played when a book is placed in a lectern.
 type LecternBookPlace struct{ sound }
 
+// WaxOn is a sound played when a sign is waxed.
+type WaxOn struct{ sound }
```
Comment (Sandertv) [inline server/world/sound/block.go:180]: Not a huge fan of the naming here. Maybe `SignWaxed` would be nicer?

File: server/world/sound/block.go [hunk 3]
```diff
@@ -176,6 +176,12 @@ type ComposterReady struct{ sound }
...
+
+// WaxedInteractFail is a sound played when a player tries to interact with a waxed sign.
+type WaxedInteractFail struct{ sound }
```
Comment (Sandertv) [inline server/world/sound/block.go:183]: Same for the naming here. Maybe something like `WaxedSignFailedInteraction`?

**Review by Sandertv — COMMENTED** [review]
Thanks for the PR! I have a couple of comments.

**Comment by Erkam246** [comment]
Done

**Pull request merged**